x-pack/plugins/lens/public/app_plugin/mounter.tsx

```diff
@@ -117,9 +116,6 @@ export async function mountApp(
   const embeddableEditorIncomingState = stateTransfer?.getIncomingEditorState(APP_ID);
 
   addHelpMenuToAppChrome(coreStart.chrome, coreStart.docLinks);
-  coreStart.chrome.docTitle.change(
-    i18n.translate('xpack.lens.pageTitle', { defaultMessage: 'Lens' })
-  );
 
```

Comment:
I've discussed with the team and the actual fix for this will be addressed elsewhere, as there are many distinct flows to take into account.

Can you restore these changes to the original state so that we can merge the PR?

Reply:
No worries, I just reverted those changes.